charts/function-mesh-operator/Chart.yaml

```diff
@@ -1,6 +1,9 @@
 apiVersion: v2
 name: function-mesh-operator
 description: function mesh operator Helm chart for Kubernetes
+maintainers:
+  - name: snbot
```

Comment:
Should we use a more formal name?

Reply:
we may add you and me as the maintainers here, but we must have valid GitHub handles. ref: https://github.com/helm/chart-testing/issues/192

Reply:
I have disabled `validate-maintainers` validation.